src/main/java/org/broadinstitute/hellbender/tools/sv/Pedigree.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.hellbender.tools.sv;
+
+import java.io.*;
+import java.util.*;
+
+public class Pedigree {
+
+    public enum Sex {
+        MALE("1"), FEMALE("2"), UNKNOWN("other");
+
+        /// stores PED format compatible format for to/from string conversions
+        private final String pedEncode;
+        Sex(String code) {
+            this.pedEncode = code;
+        }
+
+        public static Sex fromString(String sex) {
+            return switch (sex) {
+                case "1" -> MALE;
+                case "2" -> FEMALE;
+                default -> UNKNOWN;
+            };
+        }
+
+        @Override
+        public String toString() {
+            return pedEncode;
+        }
+    }
+
+    public enum Phenotype {
+        MISSING("0"), UNAFFECTED("1"), AFFECTED("2");
+
+        /// stores PED format compatible format for to/from string conversions
+        private final String pedEncode;
+        Phenotype(String code) {
+            this.pedEncode = code;
+        }
+
+        public static Phenotype fromString(String affected) {
+            return switch (affected) {
+                case "-9", "0" -> MISSING;
+                case "1" -> UNAFFECTED;
+                case "2" -> AFFECTED;
+                default -> throw new IllegalArgumentException("Invalid affected status: " + affected);
+            };
+        }
+
+        @Override
+        public String toString() {
+            return pedEncode;
+        }
+    }
+
+    public record Relationship(
+            String familyId,
+            String individualId,
+            String fatherId,
+            String motherId,
+            Sex sex,
+            Phenotype phenotype
+    ) {
+        @Override
+        public String toString() {
+            return toString("\t");
+        }
+
+        public String toString(String delimiter) {
+            return String.join(delimiter,
+                    familyId,
+                    individualId,
+                    fatherId,
+                    motherId,
+                    sex.toString(),
+                    phenotype.toString()
+            );
+        }
+    }
+
+    private final Map<String, Relationship> trios = new HashMap<>();
+
+    public Pedigree() { }
+
+    public Pedigree(String pedFilename) throws Exception
```

Comment:
In Htsjdk/Picard/GATK we're trying to move away from Strings as file identifiers.  In general we want new code to use HtsPath to identify files.  It's probably easier in this case to use Path instead though.   You can create paths from strings correctly by using IOUtils.getPath(). This allows you to read and write to things like google buckets directly.